core/src/main/java/jenkins/widgets/RunListProgressiveRendering.java

```diff
@@ -66,8 +61,11 @@ public void setBuilds(Iterable<? extends Run<?,?>> builds) {
             synchronized (this) {
                 results.add(element);
             }
-            decay *= (1 - 1 / MAX_LIKELY_RUNS);
-            progress(1 - decay);
+            progress(runs / MAX_RUNS);
```

Comment:
If the number of actual runs are less than 100 the progress will never get to 100%

Reply:
I think the progress disappears once fully loaded